File: DAGF-AlgortimoProgramacaoDeComputadores-exercicios/cap06/maior.c
#include <stdio.h>

int main(){

	int a,b,maior,menor;

	printf("WADAJU SOFTWARE SOLUTIONS\n");
	printf("Programa Maior\n");
	printf("Digite dois números inteiros\n");
	scanf("%d",&a);
	scanf("%d",&b);

	if (a > b){
		maior = a;
		menor = b;
	} else {
		maior = b;
		menor = a; 
	}
	
	printf("Números digitados A = %d e B = %d\n", a,b);
	printf("Maior = %d\n", maior);
	printf("Menor = %d\n", menor);
	printf("\nA casa tem porta!\n");

	return 0;

}

File: DAGF-AlgortimoProgramacaoDeComputadores-exercicios/cap06/maior.java
import java.util.*;

public class maior{
	public static void main(String[] args){

	int a,b,maior,menor;

	Scanner entrada = new Scanner(System.in);

	System.out.println("WADAJU SOFTWARE SOLUTIONS");
	System.out.println("Maior e menor");
	System.out.println("Digite dois números inteiros:");
	a = entrada.nextInt();
	b = entrada.nextInt();

	if (a > b){
		maior = a;
		menor = b;
	} else {
		maior = b;
		menor = a;
	}

	System.out.println("Os números digitados são A = " + a + " B = " + b);
	System.out.println("Maior = " + maior);
	System.out.println("Menor = " + menor);
	System.out.println();
	System.out.println("O programa tem porta!");

	}
}
